Libraries/Opc.Ua.Client.ComplexTypes/Opc.Ua.Client.ComplexTypes.csproj

```diff
@@ -26,6 +26,7 @@
 
   <ItemGroup Condition="'$(TargetFramework)' == 'net462'">
     <Reference Include="System.Reflection" />
+    <Reference Include="System.Numerics" />
```

Comment:
I don't see changes to ComplexTypes, why do we need a new reference?

Reply:
fewer dependencies in core